### PR TITLE
10.0 mig quality control

### PR DESCRIPTION
**Pull request by SimoRubi**
Building on #198, I have migrated this module to v10.
Please check

### CODE REVIEW

**Review by eLBati — APPROVED** [review]
After https://github.com/OCA/manufacture/pull/198

File: quality_control/data/quality_control_data.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2010 NaN Projectes de Programari Lliure, S.L.
+    Copyright 2014 Serv. Tec. Avanzados - Pedro M. Baeza
+    Copyright 2014 Oihane Crucelaegui - AvanzOSC
+    Copyright 2017 Eficent Business and IT Consulting Services S.L.
+    Copyright 2017 Simone Rubino - Agile Business Group
+    License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <data noupdate="1">
```
Comment (tafaRU) [inline quality_control/data/quality_control_data.xml]: please use `<odoo noupdate="1">` instead

File: quality_control/demo/quality_control_demo.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data noupdate="1">
```
Comment (tafaRU) [inline quality_control/demo/quality_control_demo.xml]: same as above

**Review by tafaRU — APPROVED** [review]

**Comment by SimoRubi** [comment]
@eLBati #198 has been merged

File: quality_control/README.rst [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+This module provides a generic infrastructure for quality tests. The idea is
+that it can be later reused for doing quality inspections on production lots
+or any other area of the company.
+
+Configuration
+=============
+
+* Question: The thing to be checked. We have two types of questions:
+
+ * Qualitative: The result is a description, color, yes, no...
```
Comment (LoisRForgeFlow) [inline quality_control/README.rst]: remove initial space

File: quality_control/README.rst [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+ * Quantitative: The result must be within a range.
```
Comment (LoisRForgeFlow) [inline quality_control/README.rst]: remove initial space

**Review by LoisRForgeFlow — COMMENTED** [review]
I realized after merging in v9 that the reamde was not being displayed properly. Could you fix it here?

File: quality_control/README.rst (hunk 3 above)
Comment (SimoRubi) [reply quality_control/README.rst]: thanks, fixed

File: quality_control/README.rst (hunk 4 above)
Comment (SimoRubi) [reply quality_control/README.rst]: thanks, fixed

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot + code review :+1:

**Pull request merged**